lib/file-utilities.js

```diff
@@ -35,17 +35,17 @@ const globalExclude = [
 ];
 
 // Variables
-var firstrun = true;
-var files = [];
-var tails = [];
+let firstrun = true;
+let files = [];
+let tails = [];
```

Comment:
These get re-assigned?  As long as you've gone through all of these to check for `const` capability.